jest-puppeteer.config.js

```diff
@@ -1,3 +1,14 @@
+const path = require('path');
+const dotenv = require('dotenv');
+
+dotenv.config({path: path.join(__dirname, '.env.development')});
```

Comment:
in `development-proxy` i added a script that loads the environment files in the same way as react-scripts so this will work in other environments like `test`